### ISSUE TITLE
[anchor-has-content] with a image with `alt` fails but it should pass

### ISSUE THREAD

**Issue by vhoyer** [vue-a11y/eslint-plugin-vuejs-accessibility#48]
[anchor-has-content] with a image with `alt` fails but it should pass because the alt text serves as content for the anchor
